packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -106,6 +108,32 @@ const messages = {
     `Releases ${formatReleaseDate({ date: releaseDate, withHour: true })}`
 }
 
+const renderMood = (mood: Mood) => {
+  return (
+    <TextLink
+      to={{
+        pathname: generatePath(SEARCH_PAGE, { category: 'tracks' }),
+        search: new URLSearchParams({ mood }).toString()
+      }}
+    >
+      {mood in moodMap ? moodMap[mood] : mood}
+    </TextLink>
+  )
+}
+
+const renderGenre = (genre: string) => {
+  return (
+    <TextLink
+      to={{
+        pathname: generatePath(SEARCH_PAGE, { category: 'tracks' }),
```

Comment:
yeah @Kyle-Shanks it might be nice to create a way to generate to "to" variable cleanly for search page, this is a lot of config otherwise.

Reply:
yea i agree. i don't think that we'll need to do this too often, but ill add a ticket to make a helper for this type of thing 👍